lib/api/holdings_api.rb

```diff
@@ -22,16 +22,17 @@ class HoldingsAPI < Sinatra::Base
 
     organization = params["organization"]
     item_id = params["item_id"]
-
     ht_item = Clusterable::HtItem.find(item_id: item_id)
     overlap_record = Overlap::ClusterOverlap.overlap_record(organization, ht_item)
 
     return_doc = {
+      "brlm_count" => ht_item.cluster.access_counts[organization],
```

Comment:
This is the main change, adding `brlm_count` to the return doc, regardless of any `constraint` parameter.

Reply:
Yep, old thinking on my behalf, I keep forgetting that the overlap classes exist. Fixed.